compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/NavigationDrawer.kt

```diff
@@ -933,11 +937,70 @@ private fun GraphicsLayerScope.calculatePredictiveBackScaleY(
     }
 }
 
+/**
+ * Registers a [PredictiveBackHandler] and provides animation values in [DrawerPredictiveBackState]
+ * based on back progress.
+ *
+ * @param drawerState state of the drawer
+ * @param content content of the rest of the UI
+ */
 @Composable
-internal expect fun DrawerPredictiveBackHandler(
+internal fun DrawerPredictiveBackHandler(
```

Comment:
Is there a change in AOSP for this? (I haven't [found](https://github.com/androidx/androidx/blob/androidx-main/compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/NavigationDrawer.kt#L649) in androidx-main)

We should aim to do changes there first, and only then cherry-pick it to the fork, not the other way around (commit to the fork and upstream to AOSP).

Reply:
Agree in general, but not yet, I'll do it in AOSP too. But I'd merge it in the fork first to fix the issue - we need to fix non-Android implementations, and there is no reason for copy-paste, it's easier to just make it right.

Reply:
Reverted: it's copy-paste now

Reply:
https://android-review.googlesource.com/c/platform/frameworks/support/+/3439260